src/test/java/benchmark/TwitterBenchmark.java

```diff
@@ -128,8 +132,14 @@ protected Optional<Object> getPersistedQueryId(ExecutionInput executionInput) {
         }
     }
 
-    public static void main(String[] args) {
-        ExecutionResult result = execute();
-        System.out.println(result);
+    public static void main(String[] args) throws RunnerException {
+        Options opt = new OptionsBuilder()
+                .include("benchmark.TwitterBenchmark")
+                .forks(1)
+                .warmupIterations(5)
+                .measurementIterations(10)
+                .build();
+
+        new Runner(opt).run();
```

Comment:
question - I noticed you put `main()` methods for running the JMH code.  Are you some how running this in a standard way (say on a controlled spec machine say?) ??

I ask because we have a desire to do this some how (we worked with Twitter on this until Elon Husk wrecked the joint)

Reply:
This makes it easier to iterate from the IDE and profile, setting the forks to `0`.